rpc/src/v1/impls/eth.rs

```diff
@@ -642,37 +629,14 @@ impl<C, SN: ?Sized, S: ?Sized, M, EM> Eth for EthClient<C, SN, S, M, EM> where
 	}
 
 	fn compile_lll(&self, _: String) -> Result<Bytes, Error> {
-		rpc_unimplemented!()
+		Err(errors::deprecated("Compilation of LLL via RPC is deprecated".to_string()))
 	}
 
 	fn compile_serpent(&self, _: String) -> Result<Bytes, Error> {
```

Comment:
When deprecating RPC methods, marking them as `@deprecated` in the trait definition will help us keep the wiki in sync, since the markdown builder checks for those. [Example](https://github.com/ethcore/parity/blob/ed09a76c918077e7ef9e5bdfe557fff200701eef/rpc/src/v1/traits/parity_set.rs#L77).